mappings/net/minecraft/villager/VillageDoor.mapping

```diff
@@ -1,19 +1,37 @@
 CLASS wf net/minecraft/villager/VillageDoor
 	FIELD a position Lcn;
 	FIELD c facing Lcu;
+	FIELD d lastTickSeenByVillager I
+	FIELD f entityInsideTicks I
```

Comment:
This one is difficult to name. Each entity "inside" the door will tick it, so if there are two villagers inside the door they will tick that door at twice the rate of one villager.